README.md

```diff
@@ -978,3 +978,86 @@ $ rsconnect bootstrap --server https://connect.example.org:3939 --jwt-keypath /p
 
 A full description on how to use `rsconnect bootstrap` in a provisioning workflow is provided in the Connect administrator guide's 
 [programmatic provisioning](https://docs.posit.co/connect/admin/programmatic-provisioning) documentation.
+
+## Server Administration Tasks
+
+Starting with the 2023.04 edition of Posit Connect, `rsconnect-python` can be
+used to perform certain server administration tasks, for instance managing
+runtime caches. For more information on runtime caches in Posit Connect, see the
+Connect Admin Guide's section on [runtime caches](TODO:Link).
+
+Examples in this section will use `--name myserver` to stand in for your Connect
+server information. See [Managing Server
+Information](#managing-server-information) above for more details.
+
+### Enumerate Runtime Caches
+
+Use the command below to enumerate runtime caches on a Connect server. The
+command will output a JSON object containing a list of runtime caches . Each
+cache entry will contain the following information:
+
+- `language`: The language of content that uses the cache, either R or Python.
+- `version`: The language version of the content that uses the cache.
+- `image_name`: The execution environment of the cache. The string `Local`
+  denotes native execution. For Connect instances that use off-host execution,
+  the name of the image that uses the cache will be displayed.
+
+```bash
+rsconnect system caches list --name myserver
+{
+  "caches": [
+    {
+      "language": "R",
+      "version": "3.6.3",
+      "image_name": "Local"
+    },
+    {
+      "language": "Python",
+      "version": "3.8.12",
+      "image_name": "Local"
+    },
+        {
+      "language": "R",
+      "version": "3.6.3",
+      "image_name": "teapot"
```

Comment:
Let's stop using `teapot` in real examples. Maybe use one of the r-base images? https://hub.docker.com/r/rstudio/r-base